mozci/push.py

```diff
@@ -1302,8 +1317,8 @@ def _map_failing_tasks(groups):
 
         intermittent_failures_to_be_retriggered = (
             set(g.name for g in push_groups if g.status != Status.PASS)
-            & groups_low.union(groups_no_confidence)
-        ) - groups_non_relevant_failure
+            & groups_low.union(groups_no_confidence).union()
+        ) - groups_non_consistent_failure
         intermittent_groups_still_running = groups_still_running | {
             group_name
             for group_name in intermittent_failures_to_be_retriggered
```

Comment:
this will take some logic to figure out- in here we look for is running, but if it is a retrigger or a confirm-failure, it will need to account for both.

Reply:
It is using `MOZHARNESS_TEST_PATHS` to figure out which groups (i.e. manifests) are running in a task. Is `MOZHARNESS_TEST_PATHS` set on confirm-failure tasks?

Reply:
yes, it is the full path, not the manifest name, i.e. `"MOZHARNESS_TEST_PATHS": "{\"mochitest-plain\": [\"dom/html/test/test_bug1166138.html\"]}"`

Reply:
Oh, we will have to adjust this either in mozci or in mozilla-central. In other tasks, MOZHARNESS_TEST_PATHS contains manifest names.
Could we change confirm-failures to store manifests in MOZHARNESS_TEST_PATHS like other tasks do and the test paths in another env variable?

Reply:
I would need a method for determining which test file to run since I am not running a manifest but a single test.  For test-verify, we have a patch with changed files and we look for matching tests paths in those changed files;  I am not thinking of a quick and easy way to solve this in confirm-failures; possibly another env variable.

Reply:
I have patches up for review to leave `MOZHARNESS_TEST_PATHS` alone